src/signals/IncidentMap/components/IncidentLayer.test.tsx

```diff
@@ -53,16 +50,6 @@ describe('IncidentLayer', () => {
     )
   })
 
```

Comment:
De test voor de markers is verwijderd, maar er is geen nieuwe in de plaats gekomen voor de markerClusters. Misschien goed om iets toe te voegen?

Reply:
dat is een lastig punt. als je suggesties hebt om dat voor elkaar te krijgen, graag